src/core/services/targeting-service.ts

```diff
@@ -47,6 +47,7 @@ export interface TargetingData {
 	npa?: string;
 	off?: string;
 	original_host?: string;
+	pa_available?: string;
```

Comment:
this would be a number in this case
```suggestion
	pa_available?: number;
```